app/routes/_gcn.circulars.$circularId.$version[.]json.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { LoaderFunctionArgs } from '@remix-run/node'
+import invariant from 'tiny-invariant'
+
+import { formatCircularJson } from './_gcn.circulars/circulars.lib'
+import { get } from './_gcn.circulars/circulars.server'
+import { feature, origin } from '~/lib/env.server'
+import { getCanonicalUrlHeaders } from '~/lib/headers.server'
+
+export async function loader({
+  params: { circularId, version },
+}: LoaderFunctionArgs) {
+  invariant(circularId)
+  if (!feature('CIRCULAR_VERSIONS') && version)
+    throw new Response(null, { status: 404 })
+  invariant(version)
```

Comment:
Let's rename this route to `_gcn.circulars.$circularId.($version)[.]json.ts` so that the version route parameter is optional.

Reply:
I had that previously, but this doesn't behave as expected. 

The path /circulars/####.json renders as:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/810ef202-77ff-4cfb-ae60-a528f8473808)

Reply:
This only happens if there is not  _gcn.circulars.$circularId[.]json.ts too